Marlin/ultralcd.cpp

```diff
@@ -4034,7 +4034,7 @@ void kill_screen(const char* lcd_msg) {
 
         lcd_quick_feedback();
         lcdDrawUpdate = LCDVIEW_REDRAW_NOW;
-        return_to_status_ms = millis() + LCD_TIMEOUT_TO_STATUS;
+        static millis_t return_to_status_ms = millis() + LCD_TIMEOUT_TO_STATUS;
```

Comment:
This only prevents a compile error. The actual issue is that this variable is a static local to `lcd_update` and can't be altered outside of that function without being made a general global. To prevent having to generate yet another global, one possible alternative patch is this:
```cpp
inline void handle_adc_keypad(millis_t &return_to_status_ms) {
  . . .
}

. . .

  handle_adc_keypad(return_to_status_ms);
```